spec/features/publish_a_document_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 RSpec.feature "Publishing a document" do
-  scenario "User publishes a document" do
+  scenario "User publishes a document with a lead image" do
```

Comment:
It might be best to create a new feature for this, to keep the things that are being tested per scenario to a minimum. Alternatively, we could add the publish step to the lead image feature test? That way we're end-to-end testing the workflow.

Reply:
Now added to `upload_lead_image_spec` 👍